plugins/experiment_procedure.cc

```diff
@@ -84,6 +84,15 @@ ExperimentProcedureConfigDescription()
              "Model mode: `boolean`, `regression` or `categorical`. "
              "Controls how the label is interpreted and what is the output of the classifier. "
              , CM_BOOLEAN);
+    addField("multilabelStrategy", &ExperimentProcedureConfig::multilabelStrategy,
```

Comment:
I'll write a bit more, but there is more description in the documentation of the enum and the documentation of the classifier. Should we copy the description of each enum values in the description of every field that uses it?